card/mod/05_standard/spec/set/type/signup_spec.rb

```diff
@@ -64,6 +64,7 @@
 
     it 'should be activated by an update' do
       Card::Env.params[:token] = @token
+      @signup = Card.fetch "big bad wolf"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.